docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4745,6 +4830,10 @@ message PacketMetadata {
 
 ## Master Arbitration Update
 
+P4Runtime's master arbitration mechanism ensures the following invariant: the
```

Comment:
This doesn't make a lot of sense. Can we replace it with something like this:

P4Runtime's master arbitration mechanism ensures that only the current master can modify state on the switch, and that the election id is monotonically increasing. For example, the switch must finish all previous write operations before changing masters, and must only accept write requests from the current master.

Reply:
Done.